### DISCUSSION TITLE
deleteUser()  Example

### DISCUSSION THREAD

**Discussion by FrancescoIoppolo** [capawesome-team/capacitor-firebase#472] [Q&A]
You can see an example of using the deleteUser() function
it always returns me: **No user is signed in.**
Thanks

**Comment by robingenz** [comment]
Thank you for your request.
Please provide a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) using [this template](https://github.com/capawesome-team/.capacitor-app) in a public GitHub repository so I can take a closer look.
